x11-terms/kitty-terminfo/kitty-terminfo-0.16.0.ebuild

```diff
@@ -0,0 +1,47 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{6,7,8} )
+
+inherit python-single-r1 toolchain-funcs xdg
+
+DESCRIPTION="Terminfo for kitty, an OpenGL-based terminal emulator"
+HOMEPAGE="https://github.com/kovidgoyal/kitty"
+SRC_URI="https://github.com/kovidgoyal/kitty/releases/download/v${PV}/kitty-${PV}.tar.xz"
+S="${WORKDIR}/kitty-${PV}"
+
+LICENSE="GPL-3"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="debug"
+
+REQUIRED_USE="${PYTHON_REQUIRED_USE}"
+
+RDEPEND="${PYTHON_DEPS}"
+
+DEPEND="${RDEPEND}"
+
+BDEPEND="virtual/pkgconfig"
```

Comment:
This didn't seem to install any pkgconfig files, is this dep really needed, then?

Reply:
That's right, I've done some test and it's not being used